api/src/main/java/org/pmiops/workbench/utils/mappers/CommonMappers.java

```diff
@@ -53,4 +74,92 @@ public static String cdrVersionToEtag(int cdrVersion) {
   public static int etagToCdrVersion(String etag) {
     return Etags.toVersion(etag);
   }
+
+  /////////////////////////////////////////////////////////////////////////////
+  //                                  ENUMS                                  //
+  /////////////////////////////////////////////////////////////////////////////
+
+  public static Authority authorityFromStorage(Short authority) {
```

Comment:
optional: Most of these are not really shared across models. Would it make more sense to just define them in the relevant mappers as needed instead? e.g. moving this into ProfileMapper. Or maybe just leave the truly shared "common" enums here (maybe DataAccessLevel).